manifests/splunk-kubernetes-metrics/daemonset.yaml

```diff
@@ -57,6 +57,8 @@ spec:
           requests:
             cpu: 200m
             memory: 300Mi
+        securityContext:
```

Comment:
you don't have to change files in `manifest/` folder. 
I update these when I release a new version. :)

Reply:
That works for me.